StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1005CodeFixProvider.cs

```diff
@@ -46,29 +48,60 @@ public override FixAllProvider GetFixAllProvider()
                     continue;
                 }
 
-                SyntaxTrivia trivia = root.FindTrivia(diagnostic.Location.SourceSpan.Start, findInsideTrivia: true);
-                if (!trivia.IsKind(SyntaxKind.SingleLineCommentTrivia))
+                context.RegisterCodeFix(CodeAction.Create(SpacingResources.SA1005CodeFix, t => GetTransformedDocumentAsync(context.Document, diagnostic.Location, context.CancellationToken), equivalenceKey: nameof(SA1005CodeFixProvider)), diagnostic);
```

Comment:
:exclamation: You are passing the wrong `CancellationToken` to `GetTransformedDocumentAsync`. You need to rename `t` to `cancellationToken` and pass that instead.